### DISCUSSION TITLE
is there any feature in httpx to grep multiple keywords in single http response

### DISCUSSION THREAD

**Discussion by AshiqurEmon** [projectdiscovery/httpx#1065] [Q&A] [answered]
is there any feature in httpx to  grep multiple keywords in response

like 
echo 'domain.com'|httpx -mr admin,dashboard,api

**Comment by ehsandeep** [comment, marked as answer]
@AshiqurEmon it should be possible using `-mdc` option after fixing https://github.com/projectdiscovery/httpx/issues/1019
